### ISSUE TITLE
Where do you use PCA whitening?

### ISSUE THREAD

**Issue by barry41409** [yihui-he/Residual-neural-network#1]
In your architecture fig, you use PCA whitening and Kmens to pre-process data.
Where do you use PCA whitening and K-means in your file?
I use sklearn.decomposition.PCA training CNN model, but the accuracy is too low.
https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html

**Comment by ethanhe42** [comment]
https://github.com/yihui-he/Single-Layer-neural-network-with-PCAwhitening-Kmeans
